operators/katib-controller/reactive/katib_controller.py

```diff
@@ -0,0 +1,392 @@
+                                    }
+                                ],
+                                "failurePolicy": "Ignore",
+                                "clientConfig": {
+                                    "service": {
+                                        "name": hookenv.service_name(),
+                                        "namespace": namespace,
+                                        "path": "/mutate-pods",
+                                        "port": config["webhook-port"],
+                                    },
+                                    "caBundle": ca_bundle,
+                                },
+                            },
+                        ],
+                    }
+                ],
+                "validatingWebhookConfigurations": [
+                    {
+                        "name": "katib-validating-webhook-config",
+                        "webhooks": [
+                            {
+                                "name": "validating.experiment.katib.kubeflow.org",
+                                "rules": [
+                                    {
+                                        "apiGroups": ["kubeflow.org"],
+                                        "apiVersions": ["v1beta1"],
+                                        "operations": ["CREATE", "UPDATE"],
+                                        "resources": ["experiments"],
+                                        "scope": "*",
+                                    }
+                                ],
+                                "failurePolicy": "Fail",
+                                "sideEffects": "Unknown",
+                                "clientConfig": {
+                                    "service": {
+                                        "name": hookenv.service_name(),
+                                        "namespace": namespace,
+                                        "path": "/validate-experiments",
+                                        "port": config["webhook-port"],
+                                    },
+                                    "caBundle": ca_bundle,
+                                },
+                            }
+                        ],
+                    }
+                ],
+            },
+            "configMaps": {
+                "katib-config": {
+                    f: Path(f"files/{f}.json").read_text()
+                    for f in (
+                        "metrics-collector-sidecar",
+                        "suggestion",
+                        "early-stopping",
+                    )
+                },
+                "trial-template": {
+                    f + suffix: Path(f"files/{f}.yaml").read_text()
+                    for f, suffix in (
+                        ("defaultTrialTemplate", ".yaml"),
+                        ("enasCPUTemplate", ""),
+                        ("pytorchJobTemplate", ""),
```

Comment:
Since we changed file names, should we update the suffix here? 
```suggestion
                        ("enasCPUTemplate", ".yaml"),
                        ("pytorchJobTemplate", ".yaml"),
```

Reply:
@andreyvelich: Where did those filenames get changed? I see the file names without the `.yaml` suffix here:

https://github.com/kubeflow/katib/blob/1729997/manifests/v1beta1/katib-controller/trial-template-configmap.yaml#L27

Reply:
You are right @knkski, sorry.
I thought these names are related to the file names under `/katib-controller/files`.